doc/connectivity/bluetooth/bluetooth-contrib.rst

```diff
@@ -0,0 +1,41 @@
+.. _bluetooth-contrib:
+
+Contributing to the Bluetooth subsystem
+#######################################
+
+Overview
+********
+
+This page describes how one would contribute to the Zephyr Bluetooth stack.
+It is a work in progress.
+
+The first step is familiarizing yourself with the :ref:`Stack architecture <bluetooth-arch>`.
+
+.. _bluetooth-coding-style:
+
+Coding guidelines
+*****************
+
+The following describes a few of the coding conventions we use in the Bluetooth
+subsystem. See also :ref:`Zephyr's Coding Guidelines <coding_guidelines` and the
+:ref:`Zephyr Coding Style <coding_style>`.
+
+The guidelines are not set in stone, and this is a living document. It will be
+updated if enough people agree on amending or changing the rules.
+
+Unless there is a convention listed here that disagrees, the general "rule" is
+to try to follow the style of the code surrounding the change.
```

Comment:
Maybe we should also mention that the BT coding guidelines / coding style does not (or should not) overwrite any rules from the general coding guidelines / coding style.

Reply:
Yep. Will mention that.